src/app/pages/apps-old/chart-releases/chart-releases.component.ts

```diff
@@ -185,10 +185,10 @@ export class ChartReleasesComponent implements AfterViewInit, OnInit {
     this.ws.subscribe('chart.release.query').pipe(
       untilDestroyed(this),
     ).subscribe((evt) => {
-      const app = this.chartItems.get(evt.fields.name);
+      const app = this.chartItems.get(evt.fields?.name);
 
       if (app && evt?.fields) {
-        this.chartItems.set(evt.fields.name, { ...app, ...evt.fields });
+        this.chartItems.set(evt.fields?.name, { ...app, ...evt.fields });
```

Comment:
I've noticed that currently, we have a chance that `set` will be called with `undefined` argument

Shouldn't we skip those which have empty `fields` object?

Reply:
Sure, lets skip undefined value